app/Http/Controllers/Project/Planning/SearchStringController.php

```diff
@@ -25,13 +46,12 @@ public function index($id_project)
         $synonyms = $project->synonyms;
 
         $searchStrings = $this->getSearchStrings($id_project);
-        dd($searchStrings);
 
-        return view('planning.search-string', compact('project', 'terms', 'synonyms', 'searchStrings'));
+        return view('search-string', compact('project', 'terms', 'synonyms', 'searchStrings'));
     }
```

Comment:
Isso aqui só vai ser necessário se for uma rota isolada. Nesse caso, como ele faz parte do `OverallController`, as informações `project`, `terms`, `synonyms` e `searchStrings` deveria estar lá. Já que o `@include(...)` está dentro do `project > planning > index.blade.php`.